src/client/src/orangehrmClaimPlugin/components/ClaimExpenses.vue

```diff
@@ -52,9 +53,20 @@
   </div>
   <div class="orangehrm-bottom-container">
     <oxd-text
-      >{{ $t('claim.total_amount') }}({{ currency.name }}):{{ totalAmount }}
+      >{{ $t('claim.total_amount') }} ({{ currency.name }}) : {{ totalAmount }}
```

Comment:
```vue
<oxd-text>{{ $t('claim.total_amount', {currencyName: currency.name, totalAmount: totalAmount}) }}</oxd-text>
```

```yaml
  -
    value: "Total Amount ({currencyName}) : {totalAmount}"
    unitId: total_amount
```
use parameterize language string to define context aware text rather than text concatenation.